### PR TITLE
[funmooc] Configure search filters + fix regex to extract course ids + make fix release

### PR DESCRIPTION
**Pull request by sampaccoud**
### Changed

- Customize search filters with new categories

### Fixed

- Fix regex that extracts course ids for enrollments

### CODE REVIEW

**Review by jbpenrath — COMMENTED** [review]
Search is broken if we only configure contributors filter with a reverse_id.

File: sites/funmooc/src/backend/funmooc/settings.py [hunk 1]
```diff
@@ -205,9 +205,7 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
             "", environ_name="AUTHENTICATION_BASE_URL", environ_prefix=None
         ),
         "BACKEND": values.Value(
-            "base",
-            environ_name="AUTHENTICATION_BACKEND",
-            environ_prefix=None,
+            "base", environ_name="AUTHENTICATION_BACKEND", environ_prefix=None
```
[the remark this answers is not in the document]
Comment (sampaccoud) [reply sites/funmooc/src/backend/funmooc/settings.py:208]: Not sure it's better for the naked image :+1: It's configured in the deployment tool anyway.

File: sites/funmooc/src/backend/funmooc/settings.py [hunk 2]
```diff
@@ -353,6 +349,101 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
...
+        (
+            "richie.apps.search.filter_definitions.IndexableFilterDefinition",
+            {
+                "human_name": _("Contributors"),
+                "is_autocompletable": True,
+                "is_searchable": True,
+                "min_doc_count": 0,
+                "name": "contributors",
+                "position": 6,
+                "reverse_id": "contributors",
```
[the remark this answers is not in the document]
Comment (sampaccoud) [reply sites/funmooc/src/backend/funmooc/settings.py:443]: I changed it last minute and didn't test again! :+1:

**Comment by sampaccoud** [comment]
@jbpenrath I handled your comments :wink:

**Review by jbpenrath — APPROVED** [review]
LGTM

**Pull request merged**